conda/recipes/libcudf/conda_build_config.yaml

```
@@ -26,7 +26,7 @@ aws_sdk_cpp_version:
   - "<1.11"
 
 libarrow_version:
-  - "=12"
+  - "==12.0.1.*=hb6645e0_10_cpu"
```

Would it help to just spell out the full version here (as we only listed major before)?

```suggestion
  - "=12.0.1"
```